redash/tasks/general.py

```diff
@@ -57,3 +57,16 @@ def send_mail(to, subject, html, text):
         mail.send(message)
     except Exception:
         logger.exception('Failed sending message: %s', message.subject)
+
+
+@celery.task(
+    name="redash.tasks.sync_user_details",
+    ignore_result=True,
+    soft_time_limit=60,
+    time_limit=120,
+    # let the task expire after 45 seconds since there will be
+    # another task 15 seconds later anyway
+    expires=45,
```

Comment:
Clever!

Reply:
Yeah, it simply prevents from the queue filling up with similar tasks.